### DISCUSSION TITLE
Node does not stop

### DISCUSSION THREAD

**Discussion by ryoppippi** [community/community#30111] [Copilot Conversations]
I am using Copilot on Neovim on my M1 mac.
Recently, copilot process does not stop when I quit nvim( I can see the copilot process as "node" on the activity monitor). 
I usually notice this problem when my computer get slow and I find that multiple node processes running continuously, each consuming 3-4gb of memory

**Comment by leeguooooo** [comment]
I have the same problem.

**Comment by edpanxd** [comment]
To end the program, you should be using Ctrl + C. If you do that, it sends SIGINT, which allows the program to end gracefully, unbinding from any ports it is listening on.

See also: https://superuser.com/a/262948/48624

**Reply by zbirenbaum** [reply]
Copilot is triggered from neovim by a plugin and should exit when neovim does since it is supposed to be considered a child process. There's nothing to 'Ctrl+C' here. 

Running `killall node` on the otherhand, does work.

**Comment by loeffel-io** [comment]
ref: https://github.com/orgs/community/discussions/145709
